inc/vcf/file_structure.hpp

```diff
@@ -172,13 +172,18 @@ namespace ebi
          */
         void check_chromosome() const;
         
+        void check_chromosome_colons() const;
```

Comment:
Some methods could be renamed to make their purpose clearer.

For instance, it is hard to know if this `check_chromosome_colons` should pass when colons are in the chromosome or not. Something like `check_chromosome_contains_no_colons` could be more obvious.